config/sync/core.extension.yml

```diff
@@ -10,6 +10,7 @@ module:
   block_content: 0
   block_field: 0
   breakpoint: 0
+  browsersync: 0
```

Comment:
Hmm. One problem here is that since you don't have config split, this will get enabled on pantheon.

Reply:
What's the downside to this, other than having a module enabled that's not needed? @chris-allen

Reply:
Well, assuming things are setup properly, you'll see red text in the JS console trying to attempt to browsersync

Reply:
Ok. I'll set it up.